extensions/extensions-helm/aissemble-spark-application-chart/values-migrate.yaml

```diff
@@ -0,0 +1,57 @@
+########################################
```

Comment:
Backstory: We can't update the base SparkApplication chart in-place to support these new volume mounts without breaking existing functionality in downstream projects.  We're maintaining the future state of the chart here temporarily, and applying the content manually as part of downstream test steps.